metabase/metabase
# Static hourglass indicator when reduced motion is preferred

The rotating spinner is replaced by a static hourglass icon.

How to verify:

1. Make sure to enable [prefers-reduced-motion](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion#user_preferences)
2. Do something that takes time, e.g. ask a question, click on X-ray etc

**Before this PR**

The loading is still an arc (supposed to be spinning, as the visual cue).

**After this PR**

The loading indicator is a static hourglass icon.

![image](https://user-images.githubusercontent.com/7288/140392481-88d001d1-45db-4926-8a2a-1639bef1ee13.png)
